README.md

```diff
@@ -9,6 +13,15 @@ The environment on the pi is really weird...
 
 To run simple scripted movement:
 
-`sudo /home/hera-led/Hera_games_env/bin/python scripts/test_movement.py`
+`sudo /home/hera-led/Hera_games_env/bin/python scripts/test_movement`
+
+`sudo /home/hera-led/Hera_games_env/bin/python muck_around/test_controler_input`
+
+## How to write/add more games/simulations
+
+There are two ways to add more games. The first is to write a new scripts in `scripts/`
+and then add its name to the `scripts = ` keywords in `setup.cfg`.
 
-`sudo /home/hera-led/Hera_games_env/bin/python muck_around/test_controler_input.py`
+The better long-term way is to create a new sub-package of `hera_display_games`, and
+write the game loop in a function, then add a name and point to that function in
+the `entry_points` keyword in `setup.cfg`.
```

Comment:
how do we run such functions?

Reply:
naively just running `random_walk` results in the non-sudo errors. Running with sudo doesn't find it. I think this works:
`sudo /home/hera-led/Hera_games_env/bin/random_walk -n 3 -s 1`

Reply:
ugh. This sudo thing is gross. I don' want to do that. I just want to do `random_walk -n 3 -s 1`. I'll try to understand that more.